qiskit_experiments/library/calibration/rough_frequency.py

```diff
@@ -111,3 +117,8 @@ def __init__(
             updater=Frequency,
             auto_update=auto_update,
         )
+
+    def _attach_calibrations(self, circuit: QuantumCircuit):
+        """Adds the calibrations to the transpiled circuits."""
+        schedule = self._cals.get_schedule("x", self.physical_qubits)
```

Comment:
I didn't have a chance to review the PR before merging. My one minor comment is to wonder if this should be optional. Perhaps someone would want to calibrate the 12 gates while relying on the default 01 calibrations?